### ISSUE TITLE
Building a NextJS Project in windows fails

### ISSUE THREAD

**Issue by Silverdagger21** [denoland/deno#26179]
Failing to build a NextJS project using deno in windows.
When i use "deno task build" I get the following response : 

```
Task build next build
Ignoring signal "SIGABRT" on Windows
Ignoring signal "SIGHUP" on Windows
  ▲ Next.js 14.2.12
  - Environments: .env

   Creating an optimized production build ...
Ignoring signal "SIGABRT" on Windows
Ignoring signal "SIGALRM" on Windows
Ignoring signal "SIGHUP" on Windows
Failed to compile.

Error: The file or directory is not a reparse point. (os error 4390): readlink 'Q:\Workspace\appname\node_modules\next\dist\pages\_app.js'


> Build failed because of webpack errors

```
Project works fine in dev mode using "deno task dev".
This is a project I am migrating from node so it builds & runs correctly there.

Version: deno 2.0.0 (stable, release, x86_64-pc-windows-msvc)
v8 12.9.202.13-rusty
typescript 5.6.2

**Comment by emilkallioniemi** [comment]
I'm experiencing the same issue.

**Comment by DEAD1SIDE** [comment]
got the same problem here with a completely new  project

**Comment by nathanwhit** [comment]
Just FYI this should be fixed on the latest patch release

**Comment by fushihara** [comment]
This error occurred in deno 2.0.2, but was resolved in deno 2.0.4.
